test/WebJobs.Script.Tests/Workers/Rpc/RpcWorkerConfigTests.cs

```diff
@@ -356,6 +356,45 @@ public void ValidateWorkerDescription_Succeeds(WorkerDescription workerDescripti
             Assert.Equal(defaultExecutablePath, workerDescription.DefaultExecutablePath);
         }
 
+        [Fact]
+        public void LanguageWorker_FormatWorkerPath_EnvironmentVersionEmpty()
+        {
+            // Clear FUNCTIONS_WORKER_RUNTIME_VERSION setting
+            _testEnvironment.Clear();
+
+            RpcWorkerDescription workerDescription = new RpcWorkerDescription()
+            {
+                Arguments = new List<string>(),
+                DefaultExecutablePath = "python",
+                DefaultWorkerPath = "%FUNCTIONS_WORKER_RUNTIME_VERSION%/{os}/{architecture}",
```

Comment:
There is existing test coverage when `FUNCTIONS_WORKER_RUNTIME_VERSION` is not set. This is true in placeholder mode where `FUNCTIONS_WORKER_RUNTIME_VERSION` is not set and also windows does not have 'FUNCTIONS_WORKER_RUNTIME_VERSION`